tower-batch/tests/ed25519.rs

```diff
@@ -131,8 +146,24 @@ async fn batch_flushes_on_max_latency() {
     // flushing is happening based on hitting max_latency.
     let verifier = Batch::new(Ed25519Verifier::new(), 100, Duration::from_millis(500));
     assert!(
-        timeout(Duration::from_secs(1), sign_and_verify(verifier, 10))
+        timeout(Duration::from_secs(1), sign_and_verify(verifier, 10, None))
             .await
             .is_ok()
     );
 }
+
+#[tokio::test]
+async fn fallback_verification() -> Result<(), Report> {
+    zebra_test::init();
+
+    let verifier = Fallback::new(
+        Batch::new(Ed25519Verifier::new(), 10, Duration::from_millis(100)),
+        tower::service_fn(|item: Ed25519Item| async move { item.verify_single() }),
+    );
```

Comment:
Something like this is what we'll want to copy for redjubjub and groth16 verification.